packages/zmarkdown/README.md

```diff
@@ -1,161 +1,243 @@
-# zmarkdown server HTTP API
+# zmarkdown
 
-### Usage
+  [![NPM Version][npm-image]][npm-url]
+  [![NPM Downloads][downloads-image]][downloads-url]
+  [![Linux Build][travis-image]][travis-url]
+  [![Test Coverage][coveralls-image]][coveralls-url]
 
-* See `npm run server`
-* `pm2 monit` etc
+This sub-repository is a **HTTP Server API** to provide fast and extensible **markdown parser**. It is the current Markdown engine powering [Zeste de Savoir][zds].
 
-### Requests
-All endpoints respond to `HTTP POST` requests sending a JSON body with these keys:
+It is a collection of packages extending the [**remark**
+processor][processor] and its [**MDAST**][mdast] syntax tree, [**rehype**][rehype] (for HTML processing) and [**textr**][textr] (text transformation framework). It also provides [**MDAST**][mdast] to LaTeX compilation via [**rebber**][rebber] (and its [plugins][rebber-plugins]).
```

Comment:
It is not a collection of packages, it's just a small express server leveraging remark, rehype, textr, and all of these: https://github.com/zestedesavoir/zmarkdown/tree/master/packages